scripts/deploy-gh-pages.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+mkdir tmp
```

Comment:
Maybe it's safer way to use `mktemp` than `mkdir`, though `mktemp` is not in posix but I'm sure that the running OS is Ubuntu.